pg_search/src/postgres/customscan/pdbscan/privdat.rs

```diff
@@ -33,6 +33,7 @@ pub struct PrivateData {
     var_attname_lookup: Option<*mut pg_sys::List>,
     maybe_ff: bool,
     segment_count: usize,
+    is_partial_sort: bool,
```

Comment:
Is this only consumed by explain? If so, would it be sufficient to add it only to the planning node, without also adding it to the execution state?

Reply:
Good idea. Removed it.